docs/whatsnew/7.0.rst

```diff
@@ -22,6 +22,20 @@ By the numbers:
 * X pull requests have been merged since v6.0
 * X distinct people have contributed code
 
+.. _whatsnew-7.0-table-masked-quantity:
```

Comment:
Can we merge https://github.com/astropy/astropy/pull/16646 first so RTD would build again for `latest` before merging this one? 😬